pkg/docker/index.html

```diff
@@ -753,4 +753,33 @@ <h4 class="modal-title" id="confirmation-dialog-title"></h4>
     </div>
   </div>
 
+  <!-- DELETE IMAGE CONFIRMATION DIALOG -->
+  <div class="modal" id="delete-image-confirmation-dialog"
+       tabindex="-1" role="dialog"
+       data-backdrop="static">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h4 class="modal-title" id="delete-image-confirmation-dialog-title"></h4>
+        </div>
+        <div class="modal-body" id="delete-image-confirmation-dialog-body"
+             tabindex="-1">
+          <div></div>
```

Comment:
What's this empty `<div>` used for? It would be better with an ID if your'e adding things to it in JavaScript.